### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### Pair-portfolio
created the following files:

- HTML
- CSS
- GITIGNORE
- LINTERS

### CODE REVIEW

**Review by MahmoudBakr23 — COMMENTED** [review]
# Hi @gabrielcoder247 😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

## **Invalid Submission**

- Please perform a review to your own project using the GitHub features with inline comments. Once you do this, request another review 😊🙏🏻

_Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._ 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (gabrielcoder247) [inline README.md:50]: please add the real to your social media handles

File: README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (gabrielcoder247) [inline README.md:56]: please add the real links to your social media handles :grinning: 
it feels more professional when people could reach the author easily :grinning:

**Review by gabrielcoder247 — COMMENTED** [review]
### changes required :recycle: 
It would be great you provide a link to your social media handle :grinning:  
If you have any questions you can hit me up.

Happy coding :smile:

File: README.md (hunk 2 above)
Comment (mricanho) [reply README.md:56]: - [ ] Add "Why" he needs to make this change, when you clarify the "Why", you save much frustration on the student

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @gabrielcoder247, 

Good job so far! You are so close to finish this project🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] You need at least two inline comments in order to aprove this PR, please add one more

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+![screenshot](media/hello-microverse.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```
Comment (gabrielcoder247) [inline README.md:15]: please add the major language, framework and technologies used, if any :grinning: 
It makes your work feel professional when yous state the built with details

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @gabrielcoder247  , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**